analyzers/src/SonarAnalyzer.CSharp/Extensions/InterpolatedStringExpressionSyntaxExtensions.cs

```diff
@@ -26,6 +26,6 @@ namespace SonarAnalyzer.Extensions
     public static class InterpolatedStringExpressionSyntaxExtensions
     {
         public static string GetContentsText(this InterpolatedStringExpressionSyntax interpolatedStringExpression) =>
-            interpolatedStringExpression.Contents.JoinStr("", content => content.ToString());
+            interpolatedStringExpression.Contents.JoinStr(null, content => content.ToString());
```

Comment:
I'm not very happy with this being a common extension as the empty separator is questionable. For example in hardcoded credentials rule, we needed to replace it with something nonempty. 

To avoid this problem, can we remove the interpolated string support from this rule and create an issue to deal with it in universal way for `FindConstant` extension for all other rules?

Reply:
As we discussed, we will leave it as is as it is already used in multiple rules.